src/web/app/pages-session/session-submission-page/session-submission-page.component.html

```diff
@@ -31,11 +31,11 @@ <h4>Previewing Session as
   <div class="col-12">
     <div class="alert alert-primary" role="alert">
       <div *ngIf="loggedInUser">
-        You are submitting feedback as {{ personName }}. If you wish to link your Google account ({{ loggedInUser }}) with this user,
+        You are submitting feedback as <mark><b>{{ personName }}</b></mark>. If you wish to link your Google account ({{ loggedInUser }}) with this user,
         <a href="#" (click)="joinCourseForUnregisteredStudent(); $event.preventDefault()">click here</a>.
       </div>
       <div *ngIf="!loggedInUser">
-        You are submitting feedback as {{ personName }}. You may submit feedback for sessions that are currently open and view results without logging in.
+        You are submitting feedback as <mark><b>{{ personName }}</b></mark>. You may submit feedback for sessions that are currently open and view results without logging in.
         To access other features you need to <a href="#" (click)="joinCourseForUnregisteredStudent(); $event.preventDefault()">login using a Google account</a> (recommended).
```

Comment:
@ankita2210 can you also change `(recommended)` to `(optional)`?
@hcwong I would like to squeeze in the above change to this PR as well, FYI